### DISCUSSION TITLE
Custom Atlases

### DISCUSSION THREAD

**Discussion by aswinchari** [connectomicslab/connectomemapper3#72] [Ideas]
Hello,

Does anyone have experience with custom atlases on ConnectomeMapper3? Keen to try and use the Brainnetome atlas and was wondering which files to use as the nifty and graphml files? 

Aswin

**Comment by aswinchari** [comment]
Just to update - I tried using the attached files for a single subject and the anatomical pipeline ran but I cant find the parcellation anywhere, only Lausanne. I used the nii file as the custom nifty file and the json as the graphml file. Am I doing something wrong?


[Brainnetome.zip](https://github.com/connectomicslab/connectomemapper3/files/4785338/Brainnetome.zip)

**Comment by sebastientourbier** [comment]
Custom atlases was a feature available in CMP2 but while adopting BIDS in CMP3, we have to review a lot the way CMP is working and this feature is not working anymore for the moment.
Following again the BIDS/BIDSApps philosophy, this would consist of specifying a BIDS derivatives directory obtained for instance by smriprep / dmriprep or fmriprep. I will close this "support" issue. 
The "custom" feature is now managed at the following "new feature" issue: #41 .
